javascript/javascript2/week3/class_task/script.js

```diff
@@ -0,0 +1,60 @@
+const button1 = document.getElementById("button1");
+const button2 = document.getElementById("button2");
+const counter = document.getElementById("counter");
+button1.addEventListener("click", increment);
+button2.addEventListener("click", increment);
+
+function increment(e) {
+  if (counter.innerText) {
+    counter.innerText = Number(counter.innerText) + 1;
```

Comment:
I would suggest creating a variable and incrementing that and at the end just showing the value of that variable in the HTML.

Reply:
Ok! thanks for the hint. Next time I'll use the separate counter and not the element itself